### PR TITLE
Feat/intern 1 sign up form

### PR DESCRIPTION
**Pull request by ngqtrieu99**

### CODE REVIEW

File: bookingticketui/src/api/AuthenticateAPI.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+export const BASE_URL = 'https://localhost:7089/';
+
+export const ENDPOINTS = {
+    signup: 'sign-up',
+}
+
+export const createAuthenticateAPIEndpoint = endpoint => {
+
+    let url = BASE_URL + 'api/' + 'authenticate/' + endpoint + '/';
```
Comment (tanngocIT) [inline bookingticketui/src/api/AuthenticateAPI.jsx:11]: why do we need to conbine many string?

Comment (ngqtrieu99) [reply bookingticketui/src/api/AuthenticateAPI.jsx:11]: I intend to use it as a general URL for sign-in/sign-up. Of course, I can make it into "api/authenticate/", but I think separating "api" and "authenticate" would make things clearer that this URL is only for authenticate purposes

Comment (tanngocIT) [reply bookingticketui/src/api/AuthenticateAPI.jsx:11]: let url = `${BASE_URL}api/authenticate/${endpoint}/`;

Comment (tanngocIT) [reply bookingticketui/src/api/AuthenticateAPI.jsx:11]: no need because of the method name is createAuthenticateAPIEndpoint so this only for authenticate

**Review by tanngocIT — APPROVED** [review]

**Pull request merged**